css/flexbox-guide/demos/flex-demo/align_items.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="ru">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
```suggestion
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Мы не закрываем одиночные теги.